README.md

```diff
@@ -2,6 +2,12 @@
 
 Download [Firefox iOS](https://apps.apple.com/app/firefox-web-browser/id989804926) and [Focus iOS](https://itunes.apple.com/app/id1055677337) on the App Store.
 
+|          |                |               | |
+|----------|----------------|---------------|--|
+| **Firefox-iOS**| ![Xcode 15.4](https://img.shields.io/badge/Xcode-15.4-blue?logo=Xcode&logoColor=white) | ![Swift 5.6](https://img.shields.io/badge/Swift-5.6-red?logo=Swift&logoColor=white) | ![iOS 15.0+](https://img.shields.io/badge/iOS-15.0+-green?logo=apple&logoColor=white) |
+| **Focus-iOS**| ![Xcode 15.4](https://img.shields.io/badge/Xcode-15.4-blue?logo=Xcode&logoColor=white) | ![Swift 5.6](https://img.shields.io/badge/Swift-5.6-red?logo=Swift&logoColor=white) | ![iOS 15.0+](https://img.shields.io/badge/iOS-15.0+-green?logo=apple&logoColor=white)
+---
+
```

Comment:
🎨 We spoke on chat that all badges on the page should be in one place.

Reply:
Ok perfect @clarmso i'm going to align those to the others.